controls/cis_al2023.yml

```diff
@@ -0,0 +1,2166 @@
+      - l1_server
+    status: automated
+    rules:
+      - file_groupowner_backup_etc_gshadow
+      - file_owner_backup_etc_gshadow
+      - file_permissions_backup_etc_gshadow
+
+  - id: 6.1.9
+    title: Audit system file permissions (Manual)
+    levels:
+      - l2_server
+    status: manual
+    related_rules:
+      - rpm_verify_permissions
+      - rpm_verify_ownership
+
+  - id: 6.1.10
+    title: Ensure world writable files and directories are secured (Automated)
+    levels:
+      - l1_server
+    rules:
+      - file_permissions_unauthorized_world_writable
+      - dir_perms_world_writable_sticky_bits
+
+  - id: 6.1.11
+    title: Ensure no unowned or ungrouped files or directories exist (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - no_files_unowned_by_user
+      - file_permissions_ungroupowned
+
+  - id: 6.1.12
+    title: Ensure SUID and SGID files are reviewed (Manual)
+    levels:
+      - l1_server
+    rules:
+      - file_permissions_unauthorized_suid
+      - file_permissions_unauthorized_sgid
+
+  - id: 6.2.1
+    title: Ensure accounts in /etc/passwd use shadowed passwords (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - accounts_password_all_shadowed
+
+  - id: 6.2.2
+    title: Ensure /etc/shadow password fields are not empty (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - no_empty_passwords_etc_shadow
+
+  - id: 6.2.3
+    title: Ensure all groups in /etc/passwd exist in /etc/group (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - gid_passwd_group_same
+
+  - id: 6.2.4
+    title: Ensure no duplicate UIDs exist (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - account_unique_id
+
+  - id: 6.2.5
+    title: Ensure no duplicate GIDs exist (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - group_unique_id
+
+  - id: 6.2.6
+    title: Ensure no duplicate user names exist (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - account_unique_name
+
+  - id: 6.2.7
+    title: Ensure no duplicate group names exist (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - group_unique_name
+
+  - id: 6.2.8
+    title: Ensure root PATH Integrity (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - accounts_root_path_dirs_no_write
+      - root_path_no_dot
+
+  - id: 6.2.9
+    title: Ensure root is the only UID 0 account (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - accounts_no_uid_except_zero
+
+  - id: 6.2.10
+    title: Ensure local interactive user home directories are configured (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - accounts_user_interactive_home_directory_exists
+      - file_ownership_home_directories
```

Comment:
I think the `file_groupownership_home_directories` rule should be included as well.